cert/selfsigned_test.go

```diff
@@ -15,6 +19,16 @@ const (
 var (
 	extraIPs     = []string{"1.1.1.1", "123.123.123.1", "199.189.12.12"}
 	extraDomains = []string{"home", "and", "away"}
+	privKeyBytes = [32]byte{
+		0xb7, 0x94, 0x38, 0x5f, 0x2d, 0x1e, 0xf7, 0xab,
+		0x4d, 0x92, 0x73, 0xd1, 0x90, 0x63, 0x81, 0xb4,
+		0x4f, 0x2f, 0x6f, 0x25, 0x88, 0xa3, 0xef, 0xb9,
+		0x6a, 0x49, 0x18, 0x83, 0x31, 0x98, 0x47, 0x53,
+	}
+
+	privKey, _ = btcec.PrivKeyFromBytes(
```

Comment:
This is unused, probably left over from a rebase? And this causes the whole `btcd/btcec` dependency to be pulled in. So make sure to run `go mod tidy` again after removing this.